tomcat-users.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='utf-8'?>
+<tomcat-users>
+ <role rolename="manager-gui"/>
+ <role rolename="manager-gui"/>
+ <role rolename="manager-script"/>
+ <user username="admin" password="admin" roles="manager,manager-gui,manager-script" />
```

Comment:
What blocks something other than the intended build script from using this Tomcat manager username and password to deploy something untoward to the running Docker instance?